modules/ROOT/pages/secure-storage.adoc

```diff
@@ -0,0 +1,110 @@
+= Secure Storage Overview
+
+Anypoint Studio 7.10 and earlier versions stored the information of authenticated users in the workspace, meaning that you could have different authenticated users per workspace.
+
+Studio 7.11 introduces a Secure Storage facility to encrypt and save your credentials.
```

Comment:
I would make it clear that the Secure Storage it isn't developed by us. Maybe something like:

- Studio 7.11 now uses the Secure Storage facility provided by Eclipse to encrypt and save your credentials.
- Studio 7.11 introduces the usage of the Secure Storage facility provided by Eclipse to encrypt and save your credentials.